src/main/java/org/folio/rest/impl/OrganizationApi.java

```diff
@@ -0,0 +1,72 @@
+package org.folio.rest.impl;
+
+import static io.vertx.core.Future.succeededFuture;
+import static org.folio.config.Constants.OKAPI_URL;
+
+import java.util.Map;
+
+import javax.ws.rs.core.Response;
+
+import org.folio.config.ApplicationConfig;
+import org.folio.rest.jaxrs.model.Organization;
+import org.folio.rest.jaxrs.resource.Organizations;
+import org.folio.service.organization.OrganizationService;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+
+public class OrganizationApi extends BaseApi implements Organizations {
+
+  private final String ORGANIZATIONS_LOCATION_PREFIX = "/organizations/organizations/%s";
+
+  private final OrganizationService organizationService;
+
+  public OrganizationApi() {
+    ApplicationContext ctx = new AnnotationConfigApplicationContext(ApplicationConfig.class);
```

Comment:
Initialization Spring context every time is time consuming operation.
Could you please change Spring context initialization on:

` @Autowired
   private OrganizationService organizationService;
  
  public OrganizationApi() {
    SpringContextUtil.autowireDependencies(this, Vertx.currentContext());
  }	  `